### ISSUE TITLE
missing config NS results in obscure "failed to stat path"

### ISSUE THREAD

**Issue by jti-lanl** [pftool/pftool#81]
If you've stupidly changed (or misconfigured) your MARFSCONFIGRC file, leaving out a namespace, and you run pftool on a path in that namespace-directory (e.g. with -w 1, for LSWORK) ... then get_base_path() -> factory -> stat_item() -> expand_path_info() will fail (because no such namespace), but the error report will say "failed to stat path" (because stat_item() failed).  If you go run stat on that path, it is there (and looks fine; all xattrs good, etc).

We should at least kick out a better error message (though how to actually do that is a little tricky).

**Comment by thewacokid** [comment]
This is a problem for admins (or sneaky users who remember old paths) more than a real user-facing issue.  When we remove a namespace from the config, we cycle the FUSE daemon to ensure that users can no longer see that namespace.  For all intents and purposes, that namespace is now "deleted" from the user-visible mount.

**Comment by jti-lanl** [comment]
You're right:  if I had restarted FUSE, I would've figured what was going wrong a lot quicker.
